src/extension.ts

```diff
@@ -135,6 +135,14 @@ function toggleBazelClasspathSyncStatus(doc: TextDocument){
 }
 
 function toggleBazelProjectSyncStatus(doc: TextDocument){
+	window.showInformationMessage('Project View Updated', 'Sync Project View', 'Sync Directories Only')
```

Comment:
```suggestion
	window.showInformationMessage('Project View Updated', 'Sync Project View (recommended)', 'Sync Directories Only')
```
For new users, to give more context on the recommended path (which also includes classpath updates).

Reply:
We don't/need to bloat out this UI; and syncing the project view is not necessarily recommended over sync directories, it's contextual and the user should be aware of what they're doing when the click the button

Reply:
The UX guide recommends _linking_ to more details.